### PR TITLE
Address Dashboard feedback

### LINKED ISSUE #16: Cleanup formatting busting from PR #12
I made/pushed https://github.com/filecoin-project/filecoin-storage-providers-market/pull/12 .  

I documented the issues in the attached PDF.

[Filecoin Storage Providers Market Dashboard | Filecoin Storage Providers Market.pdf](https://github.com/user-attachments/files/18044957/Filecoin.Storage.Providers.Market.Dashboard.Filecoin.Storage.Providers.Market.pdf)

I didn't investigate how to fix this yet since ran out of time 2024-12-06, and wanted to capture this by end of the week.

### PR DESCRIPTION
**Pull request by davidgasquez**
- Added new metrics for "days meeting both SLOS" and "proportion days meeting both SLOS" in the storage providers data table
- Updated the layout to fix empty space

### CODE REVIEW

File: src/data/daily_providers_metrics.csv.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@ COPY (
 
     select
         date,
-        count(distinct provider_id)  filter (where spark_retrieval_success_rate > 0.90) as meet_retrieval_sli,
-        count(distinct provider_id)  filter (where sector_health_rate between 0.95 and 1.00) as meet_sector_health_sli,
+        count(distinct provider_id) filter (where spark_retrieval_success_rate > 0.90) as meet_retrieval_sli,
```
Comment (BigLep) [inline src/data/daily_providers_metrics.csv.sh]:
```suggestion
        count(distinct provider_id) filter (where spark_retrieval_success_rate >= 0.90) as meet_retrieval_slo,
```

I'm not sure if I have the syntax right.  

Should this be `>=` given .9 is meeting?

I think it makes sense to say "I met an SLO" but not "I met an SLI" which is why I adjusted it.

File: src/data/daily_providers_metrics.csv.sh [hunk 2]
```diff
@@ -16,8 +16,8 @@ COPY (
...
+        count(distinct provider_id) filter (where sector_health_rate between 0.95 and 1.00) as meet_sector_health_sli,
```
Comment (BigLep) [inline src/data/daily_providers_metrics.csv.sh]:
```suggestion
        count(distinct provider_id) filter (where sector_health_rate between 0.95 and 1.00) as meet_sector_health_slo,
```

Is there a reason this isn't `>= 0.95`?  Is it possible to have a sector_health_rate above 1?

File: src/index.md [hunk 3]
```diff
@@ -119,16 +119,17 @@ These are aggregate views looking at all Storage Providers on the network.
...
+
 </div>
 
 
 ### Service Class Conformance
 
 Service Classes and their corresponding Service Level Objectives are defined in [filecoin-project/service-classes](https://github.com/filecoin-project/service-classes).
 
-#### "Warm" Service Class Conformance
+#### ["Warm" Service Class Conformance
```
Comment (BigLep) [inline src/index.md]:
```suggestion
#### "Warm" Service Class Conformance
```

File: src/index.md [hunk 4]
```diff
@@ -119,16 +119,17 @@ These are aggregate views looking at all Storage Providers on the network.
...
 
-[📚 "Warm" Service Class definition](https://github.com/filecoin-project/service-classes/blob/main/service-classes/warm.md).
+[📚 Definition.
```
Comment (BigLep) [inline src/index.md]:
```suggestion
[📚 "Warm" Service Class definition](https://github.com/filecoin-project/service-classes/blob/main/service-classes/warm.md).
```

Or shrink to "Definition", but include link still?

**Review by BigLep — APPROVED** [review]
I made a couple of suggestions in line.

In the process, I tried it out and thought it looked better when centered column headers and had them not get ellipsed but rather wrap.  I pushed that change.

File: src/data/daily_providers_metrics.csv.sh (hunk 1 above)
Comment (davidgasquez) [reply src/data/daily_providers_metrics.csv.sh]: Makes sense!

File: src/data/daily_providers_metrics.csv.sh (hunk 2 above)
Comment (davidgasquez) [reply src/data/daily_providers_metrics.csv.sh]: Shouldn't be possible. We can clean that up.

File: src/index.md (hunk 4 above)
Comment (davidgasquez) [reply src/index.md]: Arrg, I left that with definition! :see_no_evil: I was playing with different ways of linking.

**Pull request merged**